doc/nvim-treesitter.txt

```diff
@@ -723,6 +748,14 @@ Type (and class) definitions and annotations.
 `TSTypeBuiltin`
 Built-in types: `i32` in Rust.
 
+							  *hl-TSTypeQualifier*
+`TSTypeQualifier`
+Qualifiers on types, e.g. `const` or `volatile` in C or `mut` in Rust.
```

Comment:
actually `@const` is cStorageClass. Where is the difference to storageclass?

Reply:
storageclass refers to how a value is actually stored at the machine level. This is not relevant in many languages, but can be a critical distinction in systems languages. Examples are `static` or `register` in C, `comptime` or `linksection` in Zig, etc. `const` should not be considered storage class because it has no bearing on where a variable is stored, it only modifies the type.

Reply:
But I saw `const` being used in `synatx/c.vim` as `cStorageClass`. I couldn't find the other one used in the syntax file.